### PR TITLE
LPS-178000 | Automation ClientExtensionThemeSvg#CanExtendSvgFileFromDM and #CanExtendSvgFileFromExternalSite

### PR DESCRIPTION
**Pull request by abhnerramos**
**Description**
- Verify that new SVG spritemap is applied on the page (CanExtendSvgFileFromDM)
- Verify that new SVG spritemap file from external site is applied on the page (CanExtendSvgFileFromExternalSite)

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-178000
https://issues.liferay.com/browse/LPS-177980

### CODE REVIEW

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by abhnerramos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+				dmDocumentFile = "spritemap_example.svg",
+				dmDocumentTitle = "spritemap_example");
+		}
+
+		task ("And given A created Theme SVG extensions using the WebDAV URL from the doc-lib/") {
+			ClientExtensionGeneral.goToRemoteAppsPortlet();
+
+			ClientExtensionGeneral.addThemeSvgSpritemap(
+				entryName = "spritemap_theme",
+				entryURL = "http://localhost:8080/webdav/guest/document_library/spritemap_example.svg");
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase:60]:
```suggestion
	var portalURL = PropsUtil.get("portal.url");

	ClientExtensionGeneral.addThemeSvgSpritemap(
				entryName = "spritemap_theme",
				entryURL = "${portalURL}/webdav/guest/document_library/spritemap_example.svg");
```

**Review by mateusralv — APPROVED** [review]
Good job! Could you make these small modifications? After that if you pass the CI you can forward 😄

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by abhnerramos** [comment]
ci:test:frontend

**Comment by abhnerramos** [comment]
ci:test:sf

**Comment by abhnerramos** [comment]
ci:test:frontend

**Comment by abhnerramos** [comment]
ci:test:relevant

**Comment by mateusralv** [comment]
ci:forward

**Comment by mateusralv** [comment]
The failures are not related 🎉 I'll forward manually...

[Testray Comparison](https://testray.liferay.com/home/-/testray/runs/compare?compare=true&view=details&statusA=2&testrayRunIdB=2242853593&testrayRunIdA=2241946397&statusB=3)

**Pull request closed without merging**